### PR TITLE
Create mysql_unsupported_version_check.md

### PR DESCRIPTION
**Pull request by rnovikovP**
https://docs.google.com/document/d/1t_YfdNkFc9q6lwQQsNgonoBbvYnk9ZwMkoRehMLBBLE/

### CODE REVIEW

File: docs/advisors/checks/mysql_unsupported_version_check.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Unsupported MySQL version
+## Description
+An unsupported MySQL version in production can lead to security vulnerabilities, bugs, and instability issues. Also there will not be any support available by the vendors for any identified issues for the fixtures.
+This check verifies the current MySQL versions and identifies if it is unsupported.
+
+## Resolution
+Upgrade MySQL.
+TBD about upgrade
```
Comment (rnovikovP) [inline docs/advisors/checks/mysql_unsupported_version_check.md:8]: @patrickbirch, can you help with Instructions about the MySQL upgrade from 5.6 to 8.0 
Current example for Mongo:https://github.com/percona-platform/portal-doc/pull/59/files

Comment (patrickbirch) [reply docs/advisors/checks/mysql_unsupported_version_check.md:8]: Upgrading from 5.6 to 8.0 is not supported. You must upgrade from 5.6 to 5.7 and then upgrade from 5.7 to 8.0.

Comment (patrickbirch) [reply docs/advisors/checks/mysql_unsupported_version_check.md:8]: We have upgrade documentation for 5.7 and 8.0. Created a ticket to ask the team to review the documents to ensure they are up-to-date.

Comment (rnovikovP) [reply docs/advisors/checks/mysql_unsupported_version_check.md:8]: Can we have here something like : 
"You can't go to 8.0 directly, you need to go to 5/7, and then 8.0. Here are the instructions for the Percona server on how to do this .... "
and the links you mentioned ^

**Review by catalinaadam — APPROVED** [review]

**Pull request merged**